ingestion-beam/bin/generate-avro-test-resources.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python3
```

Comment:
Can you remove the `.py` extension on this file to match the other executables in this directory?